### PR TITLE
Quick fix: `CLI` does not allow task downloads

### PR DESCRIPTION
**Pull request by AugustoPeres**

### CODE REVIEW

**Review by Pbarbosa92 — APPROVED** [review]

**Review by IvanPombo — COMMENTED** [review]
Are you going to still add the section for download via CLI?

**Comment by AugustoPeres** [comment]
> Are you going to still add the section for download via CLI?

Yes

**Review by Pbarbosa92 — APPROVED** [review]

File: docs/cli/tracking-tasks.md [hunk 1]
```diff
@@ -66,3 +66,14 @@ of seconds to wait for confirmation:
 # Request to kill a task with a timeout for confirmation
 $ inductiva tasks kill cmvsc9qhz5iy86f6pef8uyxqt --wait-timeout 10
 ```
+
+Finally, the Inductiva CLI allows you to download tasks using:
+
+```bash
+$ inductiva cli download task_1_id task_2_id
```
Comment (IvanPombo) [inline docs/cli/tracking-tasks.md:73]:
```suggestion
$ inductiva tasks download task_1_id task_2_id
```

File: docs/cli/tracking-tasks.md [hunk 2]
```diff
@@ -66,3 +66,14 @@ of seconds to wait for confirmation:
...
+```
+
+Additionally, we can specify specific files to download using
+`--filenames` and we can specify the output directory with
+`--output_dir`. When `--output_dir` is provided the tasks will be
+download under `output_dir/tasks_id`.
```
Comment (IvanPombo) [inline docs/cli/tracking-tasks.md:79]: With `inductiva_output` before as well, correct?

**Review by IvanPombo — APPROVED** [review]
Minor comments, else LGTM!

File: docs/cli/tracking-tasks.md (hunk 2 above)
Comment (AugustoPeres) [reply docs/cli/tracking-tasks.md:79]: No, it does not use the output_dir

Comment (IvanPombo) [reply docs/cli/tracking-tasks.md:79]: Mmm... I would expect consistency between this version and the general one in Python. Wdyt @sergiomsantos ?

Comment (AugustoPeres) [reply docs/cli/tracking-tasks.md:79]: That was the original's implementation behaviour. It was then changed after comments in this commit https://github.com/inductiva/inductiva/pull/1215/commits/9e06efce37d63da55bb95a75e9ad116c60b865b4

**Pull request merged**